Security/day01/first_part/2-DNS/Readme.md

```diff
@@ -0,0 +1,12 @@
+# DNS Lookup
+
+## Objective
+
+- To understand how the Domain Name System (DNS) works and to create a program that performs a DNS lookup.
+
+## Subject
+For this exercise, you will write a program that takes a domain name as input and returns its corresponding IP address.
+
+You can use the built-in module "socket" in Python to perform the DNS lookup.
+
+![](../../.github/assets/result.png)
```

Comment:
path can be wrong, check on main branch after merge

Reply:
path has been changed